client/test/backend.go

```diff
@@ -180,6 +181,10 @@ func (b *MockBackend) Withdraw(_ context.Context, req channel.AdjudicatorReq, su
 	ch := req.Params.ID()
 	if b.isConcluded(ch) {
 		log.Debug("withdraw: already concluded:", ch)
+		e := b.latestEvents[ch]
+		if e.Version() != req.Tx.Version {
+			return fmt.Errorf("Invalid version: expted %v, got %v", e.Version(), req.Tx.Version)
```

Comment:
Should we change `expted` -> `expected` ?

Reply:
Fixed that already. You commented on an outdated commit.